addons/stock/static/src/scss/report_stockpicking_operations.scss

```diff
@@ -0,0 +1,16 @@
+.report-header-row {
+    display: -webkit-box;
+    display: flex;
+    flex-wrap: wrap;
+    -webkit-box-pack: center;
+    justify-content: center;
+    flex-direction: row;
+    -webkit-flex-direction: row;
+    -ms-flex-wrap: wrap;
+    > div {
+        flex: 1;
+        -webkit-flex: 1;
+        -webkit-box-flex: 1;
+        min-width: 150px;
+    }
+}
```

Comment:
Before this PR, `col-auto` was the closest thing to flex box support. However, we faced an issue at the time of testing that when one of the fields has long words in it, the spacing fails miserably (it gets very elongated on the vertical axis). The only way I could figure out to remove this effect and also with `wkhtmltopdf` outdated CSS support is to add `min-width` so that it forces the fields to have a readable width for this specific report. (You can also see that this solution is suggested in our [base styling for reports](https://github.com/odoo/odoo/blob/24c79ab2d6a8929ffdd3e48fbdeef5c6901c704e/addons/web/static/src/webclient/actions/reports/report.scss#L217))

Reply:
Ah right, that's the report issue I vaguely remembered. Well, fair enough, but you may want to comment the code so people know the use case otherwise people might remove it during code cleaning/refactoring